Tema3/src/Exercitul1.java

```diff
@@ -0,0 +1,43 @@
+import javafx.scene.control.SpinnerValueFactory;
+import javafx.scene.transform.Scale;
+
+import java.io.PrintStream;
+import java.util.Scanner;
+
+/**
+ * Calculate Max
+ * Create a method getMax() with two integer (int) parameters,
+ * that returns maximal of the two numbers.
+ * Write a program that reads three numbers from the console and prints
+ * the biggest of them. Use the getMax() method you just created.
+ *
+ * @author Bogdan
+ */
+
+public class Exercitul1 {
+
+    public static void main(String[] args) {
+
+        Scanner keyboard = new Scanner(System.in);
+        System.out.println("Please input the first number: ");
+        int x = Integer.parseInt(keyboard.nextLine());
+        System.out.println("Please input the second number: ");
+        int y = Integer.parseInt(keyboard.nextLine());
+        System.out.println("Please input the third number: ");
+        int z = Integer.parseInt(keyboard.nextLine());
+        int max = Math.max(x,y);
```

Comment:
using Math.max is a bit like cheating for this exercise... You could have done:
max = x;
if (max < y) max = y;
if (max < z) max = z;